### PR TITLE
remove duplicate code

### PR DESCRIPTION
**Pull request by Parachuteuk**
**What does this PR do?**
remove duplicate code
**Why are these changes required?**
to improve the code 
**This PR has been tested by:**
- Unit Tests
- Manual Testing

**Follow up**

**Extra details**

### CODE REVIEW

File: framework/src/main/java/org/tron/core/Wallet.java [hunk 1]
```diff
@@ -1740,6 +1722,21 @@ public TransactionCapsule createShieldedTransactionWithoutSpendAuthSig(
     return transactionCapsule;
 
   }
+  
+  private void ShieldedTransactionOutput (List<ReceiveNote> shieldedReceives,
```
Comment (andelf) [inline framework/src/main/java/org/tron/core/Wallet.java]: Class method naming style matters.
should be a verbal phrase. and should be in `lowerCamelCase`

File: framework/src/main/java/org/tron/core/db/PendingManager.java [hunk 2]
```diff
@@ -31,29 +31,25 @@ public PendingManager(Manager db) {
...
-      } catch (InterruptedException e) {
-        logger.error(e.getMessage());
-        Thread.currentThread().interrupt();
-      }
+      TxIteration(tx);
     }
     dbManager.getPoppedTransactions().clear();
   }
+  
+  private void TxIteration(TransactionCapsule tx) {
```
Comment (andelf) [inline framework/src/main/java/org/tron/core/db/PendingManager.java]: ditto

File: framework/src/main/java/org/tron/core/Wallet.java (hunk 1 above)
Comment (Parachuteuk) [reply framework/src/main/java/org/tron/core/Wallet.java]: done, thanks

File: framework/src/main/java/org/tron/core/db/PendingManager.java (hunk 2 above)
Comment (Parachuteuk) [reply framework/src/main/java/org/tron/core/db/PendingManager.java]: done

**Pull request merged**